guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -53,7 +53,7 @@ ifeval::["{context}" == "{project-context}"]
 --foreman-proxy-dhcp-nameservers __172.17.13.2__ \
```

Comment:
Unrelated note, but https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600 documents which IP ranges should be used for documentation.

Reply:
Interesting, I will change IP addresses in the guides to correspond with the rule.